qa/zcash/postponed-updates.txt

```diff
@@ -0,0 +1,223 @@
+rust 1.43.1 2020-09-01
+rust 1.44.0 2020-09-01
+rust 1.44.1 2020-09-01
+zeromq 4.3.2 2020-09-01
+leveldb 1.19 2020-09-01
+leveldb 1.20 2020-09-01
+utfcpp 3.1.1 2020-09-01
+crate_aes 0.4.0 2020-09-01
+crate_aesni 0.7.0 2020-09-01
+crate_aes_soft 0.4.0 2020-09-01
+crate_arrayvec 0.4.12 2020-09-01
+crate_arrayvec 0.5.0 2020-09-01
+crate_arrayvec 0.5.1 2020-09-01
+crate_arrayref 0.3.6 2020-09-01
+crate_autocfg 0.1.7 2020-09-01
+crate_autocfg 1.0.0 2020-09-01
+crate_bigint 4.4.2 2020-09-01
+crate_bigint 4.4.3 2020-09-01
+crate_blake2b_simd 0.5.9 2020-09-01
+crate_blake2b_simd 0.5.10 2020-09-01
+crate_blake2s_simd 0.5.9 2020-09-01
+crate_blake2s_simd 0.5.10 2020-09-01
+crate_bit_vec 0.5.0 2020-09-01
+crate_bit_vec 0.5.1 2020-09-01
+crate_bit_vec 0.6.0 2020-09-01
+crate_bit_vec 0.6.1 2020-09-01
+crate_bit_vec 0.6.2 2020-09-01
+crate_block_cipher_trait 0.7.0 2020-09-01
+crate_byteorder 1.3.3 2020-09-01
+crate_byteorder 1.3.4 2020-09-01
+crate_block_buffer 0.8.0 2020-09-01
+crate_block_buffer 0.9.0 2020-09-01
+crate_block_padding 0.1.5 2020-09-01
+crate_block_padding 0.2.0 2020-09-01
+crate_c2_chacha 0.2.3 2020-09-01
+crate_c2_chacha 0.2.4 2020-09-01
+crate_cfg_if 0.1.10 2020-09-01
+crate_crunchy 0.2.1 2020-09-01
+crate_crunchy 0.2.2 2020-09-01
+crate_constant_time_eq 0.1.5 2020-09-01
+crate_crossbeam 0.7.3 2020-09-01
+crate_digest 0.9.0 2020-09-01
+crate_crossbeam_channel 0.4.0 2020-09-01
+crate_crossbeam_channel 0.4.1 2020-09-01
+crate_crossbeam_channel 0.4.2 2020-09-01
+crate_crossbeam_deque 0.7.2 2020-09-01
+crate_crossbeam_deque 0.7.3 2020-09-01
+crate_crossbeam_epoch 0.8.0 2020-09-01
+crate_crossbeam_epoch 0.8.1 2020-09-01
+crate_crossbeam_epoch 0.8.2 2020-09-01
+crate_crossbeam_utils 0.7.0 2020-09-01
+crate_crossbeam_utils 0.7.1 2020-09-01
+crate_crossbeam_utils 0.7.2 2020-09-01
+crate_crossbeam_queue 0.2.0 2020-09-01
+crate_crossbeam_queue 0.2.1 2020-09-01
+crate_crossbeam_queue 0.2.2 2020-09-01
+crate_crossbeam_queue 0.2.3 2020-09-01
+crate_crypto_api_chachapoly 0.3.0 2020-09-01
+crate_crypto_api_chachapoly 0.4.0 2020-09-01
+crate_crypto_api_chachapoly 0.4.1 2020-09-01
+crate_crypto_api_chachapoly 0.4.2 2020-09-01
+crate_directories 2.0.0 2020-09-01
+crate_directories 2.0.1 2020-09-01
+crate_directories 2.0.2 2020-09-01
+crate_directories 3.0.0 2020-09-01
+crate_getrandom 0.1.13 2020-09-01
+crate_getrandom 0.1.14 2020-09-01
+crate_hex 0.4.0 2020-09-01
+crate_hex 0.4.1 2020-09-01
+crate_hex 0.4.2 2020-09-01
+crate_log 0.4.9 2020-09-01
+crate_log 0.4.10 2020-09-01
+crate_futures 0.2.0 2020-09-01
+crate_futures 0.2.1 2020-09-01
+crate_futures 0.3.0 2020-09-01
+crate_futures 0.3.1 2020-09-01
+crate_futures 0.3.2 2020-09-01
+crate_futures 0.3.3 2020-09-01
+crate_futures 0.3.4 2020-09-01
+crate_futures 0.3.5 2020-09-01
+crate_generic_array 0.13.0 2020-09-01
+crate_generic_array 0.13.1 2020-09-01
+crate_generic_array 0.13.2 2020-09-01
+crate_generic_array 0.14.0 2020-09-01
+crate_generic_array 0.14.1 2020-09-01
+crate_generic_array 0.14.2 2020-09-01
```

Comment:
How come there's so many different versions needed of each crate?

Reply:
I made it so that you have to whitelist all newer versions. It would probably be fine if I made it so you only had to whitelist the latest one but I was worried about weird cases like if we're on `3.1.0` and they release `3.1.2-hotfix` long after they release `3.2.0`.